Changelog.md

```diff
@@ -44,17 +49,44 @@ It is currently quite empty but already demonstrates some of the following scena
 
 # 5.0.0 RC
 
-* "DomainServices" dropped from all namespaces, filenames as well as nugets and DLLs.
-  * **IMPORTANT** Seach and Replace `OpenRiaServices.DomainServices` with `OpenRiaServices` in all files when uprading
+* "DomainServices" dropped from all namespaces, filenames as well as nugets and DLLs. #234
+  * **IMPORTANT** Search and Replace `OpenRiaServices.DomainServices` with `OpenRiaServices` in all files when uprading
+* Namespace `OpenRiaServices.Client.ApplicationServices` has been replaced with  `OpenRiaServices.Client.Authentication` **search and replace is needed on upgraing** #248
 * Updated required version of .Net Framework to 4.7.2 (#241)
 * Updated dependencies including EntityFramework to latests availible versions #240
-* Create, Update and Delete methods on server can now return Task #226
+  
+### Client
+
+* `IgnoreDataMemberAttribute` can now be used on the client to prevent client properties from beeing included/overwritten when loading data using `MergeIntoCurrent` or the state manipulating methods `ApplyState`, `ExtractState` etc. #249
```

Comment:
> `IgnoreDataMemberAttribute` can now be used on the client to prevent client properties from beeing included/overwritten when loading data using `MergeIntoCurrent` or the state manipulating methods `ApplyState`, `ExtractState` etc.

It would be good if we had a test that verified this new behaviour.

Reply:
I've added a tests for the new behavior